### DISCUSSION TITLE
How to disable lightning_logs folder

### DISCUSSION THREAD

**Discussion by jyh2378** [Lightning-AI/pytorch-lightning#9218] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I wonder why lightning_logs folder is generated even though I use another logger (e.g. WandB logger).
Is there some way to make generating lightning_logs folder disabled?
I want to use only logger that I select.

**Comment by rohitgr7** [comment]
just checked, it's not creating any `lightning_logs` folder.

**Reply by jyh2378** [reply]
When I just used wandb logger and callbacks.ModelCheckpoint(), ```lightning_logs``` folder is created. That means callbacks.ModelCheckpoint() is making the ```lightning_logs``` folder?

**Comment by rohitgr7** [comment]
in that case too it shouldn't create this folder and I checked and it isn't creating any. When you assign a wandb logger then model checkpoint will by default use `{logger_save_dir}/{logger_project_name}/{logger_run_id}/checkpoints` to save the checkpoints.

**Comment by jyh2378** [comment]
But, ```lightning_logs``` folder was still created.
Below is my train code.

```
if __name__=="__main__":
    wandb_logger = WandbLogger(name='version_1', project='project1', log_model=False)
    litdatamodule = LitDataModule(data_dir='../DATASETS/', batch_size=1024, num_workers=14)
    litmodel = LitModel()
    
    checkpoint_callback = callbacks.ModelCheckpoint(
        monitor="val_loss",
        dirpath="checkpoints",
        filename="model-{epoch:02d}-{val_loss:.2f}",
        save_top_k=1,
        mode="min",
    )
    
    trainer = pl.Trainer(logger=wandb_logger, gpus=[1, 2, 3, 4, 5, 6, 7], accelerator='dp', max_epochs=100, callbacks=[checkpoint_callback])
    trainer.fit(litmodel, litdatamodule)
```

Is there anything else I suspect?

**Reply by rohitgr7** [reply]
just tried https://colab.research.google.com/drive/1_04QdjoKlpsKsyhCyxbNjc8uIDPQabyf#scrollTo=p51PFkZqBZYF
no such folder is being created. Might be happening in DP mode only? can you check??
